packages/frontend/src/api/organisations.ts

```diff
@@ -20,10 +20,7 @@ interface OrganisationOutput {
   logo?: string;
 }
 
-export const getOrganisation = (
```

Comment:
Do we have to commit changes to this file?

Reply:
This will be the last commit to this file!